### ISSUE TITLE
Object Detection

### ISSUE THREAD

**Issue by LiFang522** [LiFang522/github-slideshow#2]
Can extract color histogram, color first-order moments, second-order moments and common features of grain for each image based on 900 images of wood. Parameters are optimized using both gradient descent and loss functions,color and texture features are extracted using a pre-trained model. The svm classifier is then trained based on these features and 100 sheets of labeled data.Finally, the algorithm was validated with 200 unlabeled test sets for performance, accuracy, performance, F1 metrics
